Assets/Editor/ModelInspector.cs

```diff
@@ -247,9 +288,34 @@ private void SetDisplayMode(object data)
                 case DisplayMode.Tangent:
                     OnDropDownAction(m_Settings.meshMultiPreviewMaterial, 3, false);
                     break;
+                case DisplayMode.Blendshapes:
```

Comment:
I'm testing blend shapes view on the Assets/Meshes/BlendShapesRigid.fbx/MorphBox and nothing ever renders there.